tasks/Makefile.apk

```diff
@@ -3,28 +3,32 @@ export PACKAGER_PRIVKEY = $(CURDIR)/../../key.rsa
 export APK_TEMPLATE_PATH ?= ../../apk/templates/
 export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
 export APK_PACKAGES_PATH ?= /tmp/packages
+export APK_TMP_DIR := $(shell mktemp -u -p ../../tmp -d)
 
 apk/prepare: 
-	mkdir -p $(APK_PACKAGES_PATH)
-	chmod 777 $(APK_PACKAGES_PATH)
-	cp -a $(APK_TEMPLATE_PATH)/$(APK_BUILD_TEMPLATE) APKBUILD
+	mkdir -p $(APK_PACKAGES_PATH) $(APK_TMP_DIR)
+	chmod 777 $(APK_PACKAGES_PATH) $(APK_TMP_DIR)
+	cp -a $(APK_TEMPLATE_PATH)/$(APK_BUILD_TEMPLATE) $(APK_TMP_DIR)/APKBUILD
+	cp -a . "$(APK_TMP_DIR)"
 	chsh -s /bin/sh nobody
+	chown nobody -R $(APK_TMP_DIR)
 	chmod 777 /var/cache/distfiles
-	chmod 777 .
 
 apk/checksum:
-	sudo -E -H -u nobody abuild checksum
+	cd $(APK_TMP_DIR) && \
+		sudo -E -H -u nobody abuild checksum
```

Comment:
This worries me because it the change of directory could linger.
```suggestion
		(cd $(APK_TMP_DIR) && \
		sudo -E -H -u nobody abuild checksum)
```

Reply:
it can't linger. in a `Makefile`, every line is a new shell invocation.